test/integration/scheduler_perf/util.go

```diff
@@ -134,6 +135,12 @@ func dataItems2JSONFile(dataItems DataItems, namePrefix string) error {
 
 	destFile := fmt.Sprintf("%v_%v.json", namePrefix, time.Now().Format(dateFormat))
 	if *dataItemsDir != "" {
+		// Create the "dataItemsDir" path if it doesn't exist.
+		if _, err := os.Stat(*dataItemsDir); os.IsNotExist(err) {
+			if err := os.MkdirAll(*dataItemsDir, 0750); err != nil {
```

Comment:
No need for conditional

> If path is already a directory, MkdirAll does nothing and returns nil.

https://golang.org/pkg/os/#MkdirAll

Reply:
Thanks. Updated.

Reply:
Was there a test run where the directory did not exist?

Reply:
Nope, it's just manual tests that specified non-existing dataItemsDir.